packages/enterprise/src/license.ts

```diff
@@ -260,3 +260,15 @@ export async function setLicense(l: LicenseData | null) {
   // make sure we trust that l is already verified before setting:
   licenseData = l;
 }
+
+export async function verifyCloud() {
+  if (process.env.IS_CLOUD) {
+    const CLOUD_LICENSE_KEY = process.env.CLOUD_LICENSE_KEY || "";
+    // At startup, it can take longer to fetch the public license key
+    const licenseData = await getVerifiedLicenseData(CLOUD_LICENSE_KEY, 10000);
```

Comment:
perhaps a `console.log` statement here would be helpful (`'IS_CLOUD enabled - Verifying license key...'`)